docs/docs/guides/build/components/state-backed-components/managing-state-in-production.md

```diff
@@ -0,0 +1,316 @@
+---
+title: 'Managing state in production'
+description: Learn how to refresh and manage component state in production deployments using the dg CLI.
+sidebar_position: 200
+---
+
+import Beta from '@site/docs/partials/_Beta.md';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Beta />
+
+Managing state in production depends on which state management strategy you've configured for your state-backed components. This guide explains how to refresh state for both Local Filesystem and Versioned State Storage strategies.
```

Comment:
I think this document might benefit from explaining more as to _where_ one might trigger these state updates. We explain that the `dg` command can be used, and that this can be included in CI, but should we outline other external triggers, or possible production strategies?